templates/web-application/src/index.tsx

```diff
@@ -0,0 +1,24 @@
+import "./index.css";
+
+import { StrictMode, Suspense } from "react";
+
+import { App } from "./App.tsx";
+import { Loading } from "@root/components/Loading.tsx";
+import { createRoot } from "react-dom/client";
+
+const root = createRoot(document.getElementById("root")!);
+
+// Mock Service Worker
+if (process.env.NODE_ENV === "development") {
+    import("./mocks/browser.ts").then(({ worker }) => {
+        worker.start();
```

Comment:
I would expect the mocks to be opted-in rather than bound to the environment.

A developer might want to connect his development environment to a backend API. In this scenario, would MSW let the request passthrough it would it pass the request to the mock?

Reply:
I open this issue concerning this: https://github.com/workleap/wl-foundry-cli/issues/34